crit-go/cmd/decode.go

```diff
@@ -0,0 +1,55 @@
+/*
+Copyright © 2020 NAME HERE <EMAIL ADDRESS>
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
```

Comment:
We usually do not add license headers to the code in criu or go-criu. But if you add it, you should put your name in there.

Reply:
i'll remove it :+1:

Reply:
> Strange that Travis did not run.

so travis seems to have ran now,added signed commits and proper refactored structure. Would love your feedback on the code. 

**Changes**
- License Header Removed 
- All commits changed to Signatured Commits as per contribution guidelines 
 - Stdin support added to Decode,Encode,Show,X if --in flag is blank
- For Explore Switch cases added to evaluate flags and trigger respective functions.
- For explore single what flag added which supports arguments instead of individually defined -ps ,rss,fds,mems flags and clumsy evaluation of flags was removed 
- Moved the core funcs to a separate  package instead of keeping everything n the cmd package. 

works rn [screenshot](https://imgur.com/a/r2tOfHW)